pkg/mark/renderer/blockquote.go

```diff
@@ -50,9 +51,10 @@ func (m BlockQuoteLevelMap) Level(node ast.Node) int {
 
 // ClassifyingBlockQuote compares a string against a set of patterns and returns a BlockQuoteType
 func ClassifyingBlockQuote(literal string) BlockQuoteType {
-	infoPattern := regexp.MustCompile(`info|Info|INFO`)
-	notePattern := regexp.MustCompile(`note|Note|NOTE`)
-	warnPattern := regexp.MustCompile(`warn|Warn|WARN`)
+	infoPattern := regexp.MustCompile(`(?i)info|important|caution`)
+	notePattern := regexp.MustCompile(`(?i)note`)
```

Comment:
Looking at the graphics in 
https://confluence.atlassian.com/conf59/info-tip-note-and-warning-macros-792499127.html
 and 
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

should we move caution to this one?

Reply:
You mean moving caution to be together with warning    ?

Reply:
Just looking at the symbols and colour schemes, I think this mapping is the best to support Github Alerts:
| Github | Confluence |
|---|---|
| Note (blue I in circle) | Info (blue I in circle) |
| Tip (green lightbulb)| Tip  (green checkmark in circle) |
| Important (purple exclamation mark in speech bubble | Info (blue I in circle) |
| Warning (yellow exclamation mark in triangle) | Note (yellow exclamation mark in triangle) |
| Caution (red exclamation mark in hexagon) | Warning  (red exclamation mark in hexagon) |

Reply:
Please review again after the last commit if it is matching your expectations :)